pkg/tests/end_to_end_tests/generate_jaeger_response_test.go

```diff
@@ -14,16 +14,12 @@ import (
 	"fmt"
```

Comment:
IIUC, all jaeger related tests which we own can be removed once we get jaeger storage integration tests up & running right? I feel most of the tests which we own are redundant. WDYT?

Reply:
The other point is that some of the tests we have also test the OTEL ingest pipeline. The thing is that the responses to compare against the fixtures are retrieved using the Jaeger endpoint.